sdk/eventhub/azure-eventhubs/azure/eventhub/eventprocessor/partition_manager.py

```diff
@@ -0,0 +1,44 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# -----------------------------------------------------------------------------------
+
+from typing import Iterable, Dict, Any
+from abc import ABC, abstractmethod
+
+
+class PartitionManager(ABC):
+    """Subclass PartitionManager to implement the read/write access to storage service to list/claim ownership and save checkpoint.
+
+    """
+
+    @abstractmethod
+    async def list_ownership(self, eventhub_name: str, consumer_group_name: str) -> Iterable[Dict[str, Any]]:
```

Comment:
`list_ownerships` (plural)?

Reply:
Our EventHub .NET dev Jesse says "ownership" is both singular and plural ;-)